components/search/SearchModule.jsx

```diff
@@ -164,12 +164,12 @@ class SearchModule extends React.Component {
                        src="https://storage.googleapis.com/nft-search/img/search-icon%402x.svg"/>
                   <input
                     className={this.props.page ? "desktop search-all-nfts-results" : "desktop search-all-nfts"}
-                    placeholder="Visual search from images by keywords"
```

Comment:
shouldn't this go away since we no longe use reverse image?

Reply:
this uses 
https://api.nftport.xyz/v0/recommendations/similar_nfts/urls endpoint (before was using visual_search or v0/recommendations/similar_nfts/urls)